NPM_VERSION

```diff
@@ -0,0 +1 @@
+10.2.0
```

Comment:
10.2.0? what a big jump :joy: 

please put it back to v0.2.1, the current version.

Reply:
Sorry 😅